src/frontend/src/components/Wallet.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="container">
+    <h1>{{ walletAmount }} BSC</h1>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+@Component
+export default class Wallet extends Vue {
+  get walletAmount() {
```

Comment:
Instead of accessing the state here we can just access the getter!

Reply:
Fixed